src/osd/PrimaryLogPG.cc

```diff
@@ -2636,6 +2616,10 @@ int PrimaryLogPG::do_manifest_flush(OpRequestRef op, ObjectContextRef obc, Flush
     }
   }
 
+  if (manifest_fop->io_tids.empty()) {
+    try_flush_mark_clean(manifest_fop);
```

Comment:
Specifically, try_flush_mark_clean for a cache/tier pool does a few things including clearing the dirty bit and potentially evicting the object.  Neither of those operations are obviously (to me at least) applicable here.

Reply:
What does the DIRTY object_info_t flag mean for a manifest object?
--> Now, it is meaningless.

Specifically, try_flush_mark_clean for a cache/tier pool does a few things including clearing the dirty bit and potentially evicting the object. Neither of those operations are obviously (to me at least) applicable here.
--> Right, I missed something. This needs to be removed. My intention is that we need try_flush_mark_clean because we haven't implement evict feature (which is the future work, probably a next PR).

Reply:
Ok, please do and update the relevant commits.